vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -156,22 +157,12 @@ func validateCreateOptions(r volume.Request, fsMap map[string]string) error {
 		r.Options[fsTypeTag] = fs.FstypeDefault
```

Comment:
We would still need the `fstype` in the request for the VMDK Ops Service to persist it in the KV Store. However, I could move the default `fstype` selection logic to `fs` if needed.